README.md

```diff
@@ -1,44 +0,0 @@
-# Research notes
```

Comment:
some documentation would be highly appreciated to understand how to work with this repo since readme is being dropped

Reply:
right… it's nested in :D but I'll include something, I did the same for the packit.dev